README.md

```diff
@@ -111,3 +111,38 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+Place the tool somewhere in your path, using either:
+* make install
```

Comment:
Thanks! Please use the steps in the previous comment (with `sudo` and `chmod`) though -- just plain `make install` doesn't work (it tries to install to a global location w/o sufficient permissions).